src/strings.js

```diff
@@ -1,25 +1,27 @@
 function sayHello (string) {
-  // your code here
+  return 'Hello, ' + string + '!';
```

Comment:
You can also use string interpolation here `Hello, ${string}!`

Reply:
With back ticks https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals